shared/src/mobileMain/kotlin/org/jetbrains/kotlinconf/ui/SearchScreen.kt

```diff
@@ -82,11 +82,11 @@ fun SearchScreen(
                     selectedTab = it
                 }
             )
-            Row(horizontalArrangement = Arrangement.End, modifier = Modifier.fillMaxWidth()) {
+            Row(horizontalArrangement = Arrangement.Start, modifier = Modifier.fillMaxWidth()) {
```

Comment:
It looks like it affects the mobile design as well. Should it be platform specific?

Reply:
btw. UI changes should not be related to the notification manager